docs/General/Manuals/client-manual.rst

```diff
@@ -1454,6 +1454,32 @@ Unit ID value.
 Mini Map
 --------
 
+On the `Map View`_, in the lower right corner will be a :guilabel:`Mini Map` widget. Figure 43 below shows a
+sample of what the :guilabel:`Mini Map` looks like for a single-player game. Figure 44 shows an example of
+what a :guilabel:`Mini Map` looks like for a LongTurn multi-player game. The big difference between the two
+is the single-player :guilabel:`Turn Done` button is replaced by a countdown timer. On the right hand side of
```

Comment:
"...shows ~~a sample~~ *an example* of..."

Use "singleplayer" without a dash, same for "multiplayer".

"...the Turn Done button is replaced ~~by~~ *with* a countdown timer."